### DISCUSSION TITLE
Mnet

### DISCUSSION THREAD

**Discussion by sooyaaa131995** [iptv-org/community#361] [Show and tell]
### Channel ID

Mnet.kr

### Stream URL

http://mnet.xp3.biz/mnet.m3u8

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by BellezaEmporium** [comment]
Interesting proxy usage, takes a bit of time to load but works fine.

**Comment by Alstruit** [comment]
Link is dead.

**Comment by AzeemHadzrie** [comment]
Try that: http://mytv.dothome.co.kr/ch/catv/224.php

It works on VLC.

**Comment by Alstruit** [comment]
> Try that: http://mytv.dothome.co.kr/ch/catv/224.php
> 
> It works on VLC.

That links points to seezntv. I'm planning to add dothome links to iptv-org/iptv#9648.

**Comment by AzeemHadzrie** [comment]
Alright.

**Comment by BellezaEmporium** [comment]
> Link is dead.

It was MNet Japan, uses another server, 403s if used outside of Japan.

**Comment by freearhey** [comment]
http://mnet.xp3.biz/mnet.m3u8 (offline)
<img width="238" alt="image" src="https://user-images.githubusercontent.com/7253922/232237688-47ee0a7d-cde8-4e3e-8a77-906d8272630e.png">

http://mytv.dothome.co.kr/ch/catv/224.php (offline)
<img width="329" alt="image" src="https://user-images.githubusercontent.com/7253922/232237724-dee8a54a-d7d7-452f-84ca-314713261695.png">
